upgrade-binaries/README.md

```diff
@@ -0,0 +1,12 @@
+# Static binaries for 1.x to 2.x upgrades
+
+Some additional tools are required for upgrading a 1.x resinOS prior to 1.27 to 2.x. These are:
+
+ * e2label
+ * mkfs.ext4
+ * resize2fs
+ * tar
+
+These binaries have been built with buildroot, which makes static builds much easier than Yocto.
```

Comment:
Trying to reproduce this, but the latest git from Buildroot does not have for example `BR2_PACKAGE_E2FSPROGS_E2LABEL` config option, and trying `2017.05` (as the config suggest) the config needs adjustments (make menuconfig and then save) to be able to build even, without extra configuration. Checking it more, this is some report of what I've tried.

Regarding the config, I think it would be great of there was a minimal defconfig version, that can be copied into buildroot's `configs`, and made with `make resinhup_static_defconfig && make` or something like that, this would make it reproducible, imho. Not a big deal, but bothering a bit :)